sbi/inference/abc/mcabc.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from pyro.distributions.empirical import Empirical
+from torch.distributions.distribution import Distribution
+from sbi.inference.abc.abc_base import ABCBASE
+from typing import Callable, Optional, Union, Tuple
+
+import torch
+from torch import Tensor, ones
+from numpy import ndarray
+
+
+class MCABC(ABCBASE):
+    def __init__(
+        self,
+        simulator: Callable,
+        prior,
+        x_o: Union[Tensor, ndarray],
+        distance: Union[str, Callable] = "l2",
+        num_workers: int = 1,
+        simulation_batch_size: int = 1,
+        show_progress_bars: bool = True,
+    ):
+        """Monte-Carlo Approximate Bayesian Computation (Rejection ABC).
+
+        Args:
+
+        See docstring of `ABCBASE` class for all arguments.
```

Comment:
If we want to put this in mkdocs (I guess we do?) you have to manually copy over the docstring.

Reply:
OK. I have to copy it from where to where?